docs/specs/validations/metadataContentValidations.yml

```diff
@@ -2461,3 +2461,187 @@ outputs:
   4667fedf/v2/b.json:
   .errors.log: |
     {"message_severity":"suggestion","log_item_type":"user","code":"h1-in-moniker","message":"H1s are not allowed in moniker sections. Each article should have one and only one H1.","file":"v2/b.md","line":2,"end_line":2,"column":1,"end_column":1}
+---
+# Content validation - codeblock validation indented checking
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "codeBlocks": {
+        "rules": [
+          {
+            "type": "CodeBlockIndented",
+            "message": "Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```).",
+            "code": "code-block-indented",
+            "severity": "SUGGESTION",
+            "exclusions": [
+              "includes",
+              "landingpage",
+              "hubpage"
+            ],
+            "excludedDocfxVersions": [
+              "v2"
+            ]
+          }
+        ]
+      }
+    }
+  a.md: |
+    ---
+    title: Axes object (Excel)
+    ---
+
+    :::image type="complex" source="image.png" alt-text="triple-colon test":::
+         this is an intented codeblock included in triple colon block
+    :::image-end:::
+         this is an intented codeblock
+outputs:
+  a.json:
+  .errors.log: |
+    {"message_severity":"warning","log_item_type":"user","code":"file-not-found","message":"Invalid file link: 'image.png'.","file":"a.md","line":5,"end_line":5,"column":1,"end_column":1}
```

Comment:
Can you add a file image.png in the test to aviod useless validation result in this test case?

Reply:
ok